crates/common_enums/src/enums.rs

```diff
@@ -2623,3 +2623,157 @@ mod tests {
         assert!(result.is_err());
     }
 }
+
+#[derive(Default, serde::Serialize, Debug, Clone, serde::Deserialize, strum::Display)]
+#[serde(rename_all = "snake_case")]
+#[serde(tag = "connector_name", content = "connector_account_details")]
+pub enum ConnectorAuthType {
+    #[strum(serialize = "aci")]
+    Aci {
```

Comment:
SInce this type can be used in other places, better to convert it to a common struct, and use it here as
```rust
#[serde(transparent)]
#[strum(serialize = "aci")]
Aci(AciAuthType)
```